docs/changes/utils/16224.api.rst

```diff
@@ -0,0 +1,3 @@
+Unmasked ``Masked`` scalar instances are now considered hashable, to match the
+implicit behaviour of regular arrays, where if an operation leads to a scalar,
+an hashable array scalar is returned.
```

Comment:
```suggestion
a hashable array scalar is returned.
```

Reply:
Will get to this tomorrow! For now, just pushed an extra commit so that test (hopefully) pass.

Reply:
Done.